google/datalab/ml/_fasets.py

```diff
@@ -65,7 +65,7 @@ def plot(self, data):
 class FacetsDiveview(object):
```

Comment:
Just realized that this file is named "fasets".  Don't need to actually fix now...

Reply:
Ah. Will fix in a future change.